task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,18 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    let result = '';
+    if(num % 3 === 0) {
+        result = 'Fizz';
```

Comment:
Hello Alex. Thank you for the review.
Tonight I was sitting the whole evening thinking how to return the result without additional checks.
Unfortunately, I couldn't find anything better. 
I suppose my previous solution was too misleading, so I changed it totally.
I hope you'll like it.

Reply:
I meant that you could jsut od this
```

if(num % 3 === 0) 
        return 'Fizz';
```


instead of 
```

if(num % 3 === 0) {
        result = 'Fizz';
```